bin/generate-gutenboarding-design-thumbnails.js

```diff
@@ -44,6 +75,7 @@ const getDesignUrl = ( design ) => {
 			font_base: design.fonts.base,
 			font_headings: design.fonts.headings,
 			site_title: design.title,
+			...( language && language !== 'en' && { language } ),
```

Comment:
You may need [viewport_height](https://github.com/Automattic/wp-calypso/blob/0e3a272e1446247cd98d583d5f817ce0a1398b4a/client/landing/gutenboarding/available-designs.ts#L41) here too.